### DISCUSSION TITLE
HDMI not working

### DISCUSSION THREAD

**Discussion by ThomasAn73** [wottreng/Linux-Mint-on-Lenovo-Legion-5#6] [Q&A]
Great resource. One issue is HDMI is not working on Legion 5 with Mint. Therefore no second monitor support.

**Comment by O491dogan** [comment]
HDMI is directly connected to your discrete graphics card e.g. NVIDIA. 
If you are on power saving mode (nvidia optimus) then the discrete gpu is not and cannot be used (at least I think so).

Try to set the profile to "on demand" or to "performance" and check if the monitor works.

What I also do have in mind is that  it might be the case that the monitor has to be connected at boot time. But this might be wrong.

**Comment by ThomasAn73** [comment]
I got it figured. It works good on the built-in HDMI now. Thanks again !
